cpp/include/cudf/null_mask.hpp

```diff
@@ -160,4 +160,28 @@ rmm::device_buffer copy_bitmask(column_view const& view,
     cudaStream_t stream = 0,
     rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**---------------------------------------------------------------------------*
+ * @brief Copies `views[i]`'s bitmask from the bits
```

Comment:
Does this concatenate the null masks? If so, call it `concatenate_bitmasks` and modify/use the existing gdf_mask_concat kernel, rather than a loop that runs multiple copies.